### PR TITLE
Add software-upate

### PR DESCRIPTION
**Pull request by padmashree9110**
Adding software-update plugin, Which will help to update intel-aero image.

### CODE REVIEW

**Comment by padmashree9110** [comment]
Verified fail condition also by connecting the USb with no image and without USB.

File: software-update/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+"version": 0,
+
+    "tools": {
+        "update": {
+            "label": "Software-update",
```
Comment (lucasdemarchi) [inline software-update/manifest.json]: Should be "Software Update"

File: software-update/update.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<head>
+<title>Software-update</title>
```
Comment (lucasdemarchi) [inline software-update/update.html]: "Software Update"

File: software-update/update.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
+    <script src="../base1/jquery.js"></script>
+    <script src="../base1/cockpit.js"></script>
+    <script type="text/javascript">
+
+function fun1()
+{
+            var proc = cockpit.spawn("aero-reboot-update.py");
+            proc.fail(On_fail);
+        }
```
Comment (lucasdemarchi) [inline software-update/update.html]: this function is failing parser... ?  Also check your spaces... indent should be 4 spaces.

**Review by lucasdemarchi — CHANGES_REQUESTED** [review]

File: software-update/update.html (hunk 3 above)
Comment (padmashree9110) [reply software-update/update.html]: Yes, It is parsing fail condition.

File: software-update/update.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<head>
+    <title>Software Update</title>
+    <meta charset="utf-8">
+    <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
+    <script src="../base1/jquery.js"></script>
+    <script src="../base1/cockpit.js"></script>
+    <script type="text/javascript">
+
+        function fun1() {
```
Comment (lucasdemarchi) [inline software-update/update.html:9]: `fun1`? you should name functions based on what they are doing.

**Comment by lucasdemarchi** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/intel-aero/aero-cockpit-plugins/compare/bff30c3~2...bff30c3)

**Comment by lucasdemarchi** [comment]
Changes like suggested should be amended on the commit rather than creating commits on top with fixes.
Some things missing were:

1. Add the plugin to the Makefile, so when we call "make install" it is automatically installed
1. Renames so functions and messages make more sense

Check my commits on top. Thanks.

**Pull request closed without merging**